tests/e2e/support/api/davSpaces/spaces.ts

```diff
@@ -20,24 +23,20 @@ export const folderExists = async ({
   return getResponse.status === 200
 }
 
-export const createFolder = async ({
+const createFolder = async ({
   user,
-  folder
+  folder,
+  webDavEndPathToRoot // this path only determines the end path of a webdav end point API to its root path
```

Comment:
I would find that more understandable, but choose whatever you like more
```suggestion
  webDavRootPath // the root of the WebDAV path. This is `spaces/<space-id>` for ocis or `files/<user-id>` for oC10
```